domains/certificates/Certificates.sln

```diff
@@ -25,7 +25,11 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Worker", "RegistryConnector
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Worker.UnitTests", "RegistryConnector\Worker.UnitTests\Worker.UnitTests.csproj", "{3FC7672A-5A92-4B96-94E3-60B7468148FD}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "API.IntegrationTests", "Query.API\API.IntegrationTests\API.IntegrationTests.csproj", "{741B4018-7BB1-46EF-A285-87EC11418FC5}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "API.IntegrationTests", "Query.API\API.IntegrationTests\API.IntegrationTests.csproj", "{741B4018-7BB1-46EF-A285-87EC11418FC5}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DomainCertificate", "Shared\DomainCertificate\DomainCertificate.csproj", "{1F343067-3640-4209-9D7B-33D2B2E26033}"
```

Comment:
Just from reading the name i am not sure what the purpose of this is. Can we be more specific?

Reply:
This project was added to have the shared "things" between the domain events defined in CertificateEvents-project and the messages sent with MassTransit defined in Contracts-project.
I would say that these "things" are Value Objects or primitives within our domain. So a more specific and correct name could be "ValueObjects" or "CertificatePrimitives"

Reply:
But it's not only for valueobjects and primitives. Aggregates could live there as well

Reply:
Aggregates are placed in the CertificatesEvents-project at this point. Or the only aggregate that we have is placed there.

Reply:
And why is that aggregate placed there? I think it would be better to have it in the DomainCertificate project

Reply:
DomainCertificate-project is created in this PR - it did not exist before. Before there was only CertificatesEvents-project, which given that it contained ValueObjects/records/primitives, domain events and an aggregate/write model could more aptly be named "CertificatesDomain" or "DomainCertificate". 

But the underlying issue is the desire to keep a clear separation between the communication events (MassTransit-contracts) and the domain events. In order to avoid too much duplication we wanted to move the stuff that was shared (e.g. GSRN, Period, ShieldedValue, TechCode, FuelCode) into its own project. Maybe that is just too difficult for us - and letting domain events also be communication events is easier. Not sure if it will bite us in the a.. later.

Reply:
Renamed to CertificateValueObjects